controllers/course.js

```diff
@@ -1,4 +1,5 @@
 const courseService = require('~/services/course')
+const getMatchOptions = require('~/utils/getMatchOptions')
```

Comment:
where do you use it?

Reply:
> where do you use it?

oops, forgot about that, thank you, removed it